tests/config/test_config.py

```diff
@@ -63,6 +65,16 @@ def create_config_files():
     yml.dump(DBT_PROJECT, os.path.join(FILE_DIR, 'dbt_project.yml'))
     yml.dump(PROFILES, os.path.join(FILE_DIR, 'profiles.yml'))
 
+def create_slack_workflows_config_files():
```

Comment:
Could you please pass the CONFIG or WORKFLOWS_CONFIG  to the 'create_config_files' function, instead of duplicating its code?

Reply:
[using single create config method with config type passed as param](https://github.com/elementary-data/elementary/pull/54/commits/70223b98af5fa1edbe6b915f0dc496465b883f51)